packages/console/src/pages/Applications/components/GuideCard/index.tsx

```diff
@@ -1,35 +1,63 @@
+import { conditional } from '@silverhand/essentials';
 import classNames from 'classnames';
 import { Suspense } from 'react';
 
 import { type Guide, type GuideMetadata } from '@/assets/docs/guides/types';
 import Button from '@/ds-components/Button';
+import { onKeyDownHandler } from '@/utils/a11y';
 
 import * as styles from './index.module.scss';
 
 export type SelectedGuide = {
-  target: GuideMetadata['target'];
   id: Guide['id'];
+  target: GuideMetadata['target'];
+  name: GuideMetadata['name'];
 };
 
 type Props = {
   data: Guide;
   onClick: (data: SelectedGuide) => void;
   hasBorder?: boolean;
+  isCompact?: boolean;
 };
 
 function LogoSkeleton() {
   return <div className={styles.logoSkeleton} />;
 }
 
-function GuideCard({ data, onClick, hasBorder }: Props) {
+function GuideCard({ data, onClick, hasBorder, isCompact }: Props) {
   const {
     id,
     Logo,
     metadata: { target, name, description },
   } = data;
 
+  const onClickCard = () => {
+    if (!isCompact) {
+      return;
+    }
+
+    onClick({ id, target, name });
+  };
+
+  const compactCardProps = conditional(
+    isCompact && {
+      tabIndex: 0,
+      role: 'button',
+      onKeyDown: onKeyDownHandler(onClickCard),
+      onClick: onClickCard,
+    }
```

Comment:
we can just inline this? since `compactCardProps` sounds ambiguous

Reply:
OK, np